docs/api/migration-guides/qiskit-1.0.mdx

```diff
@@ -0,0 +1,659 @@
+   pip install 'qiskit>=1' jupyterlab pandas matplotlib
+   ```
+
+   Qiskit 1.0 includes breaking changes, so several packages are marked as not-yet-compatible with it.  Therefore, you might see errors from `pip` until new versions of those packages are released. Old versions of packages might also depend on the legacy `qiskit-terra` package.  Such packages might not return errors when running this command, but might raise an error when running `import qiskit`.
+
+   <Admonition type="caution">
+   Do not install packages that are incompatible with Qiskit 1.0 on this virtual environment.  If you need to use such packages, install them in a separate virtual environment with Qiskit 0.45 or 0.46.
+   </Admonition>
+
+   If you have an existing environment, you can use [`pipdeptree`](https://github.com/tox-dev/pipdeptree/blob/main/README.md#pipdeptree) to query the requirements of your installed packages to see if they require `qiskit<1`.  For any that require `qiskit<1>`, check for updates that make it compatible with Qiskit 1.0. 
+
+   If you encounter issues, consult the [troubleshooting](#troubleshooting) section, or ask on [Qiskit Slack](https://qisk.it/join-slack). If you think there is a bug, you can [create an issue against Qiskit](https://github.com/Qiskit/qiskit/issues/new/choose).
+
+
+4. If you are not planning to use the environment immediately, use the `deactivate` command to leave it.
+
+
+<span id="activating-a-venv"></span>
+### Use the new environment
+
+Each time you start a new command line session, you must "activate" the environment by running the `activate` command:
+
+        <OperatingSystemTabs>
+          <TabItem value="mac" label="macOS">
+             ```bash
+             source /path/to/qiskit-1.0-venv/bin/activate
+             ```
+          </TabItem>
+
+          <TabItem value="linux" label="Linux">
+             ```bash
+             source /path/to/qiskit-1.0-venv/bin/activate
+             ```
+          </TabItem>
+
+          <TabItem value="win" label="Windows">
+            ```powershell
+            C:\path\to\qiskit-1.0-venv\Scripts\activate.ps1
+            ```
+          </TabItem>
+         </OperatingSystemTabs>
+
+<span id="for-developers"></span>
+## For developers
+
+If you maintain a package that depends on Qiskit, use this information to learn how to correctly express your compatibility and test against Qiskit 1.0.
+
+
+### Recommendations for requirements
+
+We recommend that your package requires `qiskit>=0.45,<1` (or other appropriate lower bound) if you are not certain whether the package is compatible with Qiskit 1.0.
+This is [the same recommendation being made for NumPy 2.0 compatibility.](https://github.com/numpy/numpy/issues/24300)
+
+A Qiskit 1.0 release candidate, version 1.0.0rc1, will be released on 1 February 2024. 
+You should test your package against this, and as soon as possible, release a new (compatible) version of your package with its upper requirement unpinned.
```

Comment:
I don't know if it is worth mentioning here that making a new release as soon as possible is important because even if a project releases a new release with `qiskit<1` as a dependency `pip` will just install the previous release that did not have the upper bound on `qiskit`. That behavior is mentioned in the troubleshooting section.

Reply:
I'll probably leave it out in the interests of "brevity" (he says about a 6k-word document).  If we try to justify everything _too_ much, we run the risk of making it harder to spot the action items, and to some degree, a package maintainer reading this is already trusting that we're making these recommendations from a position of authority.